server/graphql/common/expenses.ts

```diff
@@ -251,6 +252,19 @@ export const canReject = async (req: express.Request, expense: typeof models.Exp
   }
 };
 
+/**
+ * Returns true if expense can be rejected by user
+ */
+export const canMarkAsSpam = async (req: express.Request, expense: typeof models.Expense): Promise<boolean> => {
+  if (![expenseStatus.REJECTED].includes(expense.status)) {
+    return false;
+  } else if (!canUseFeature(req.remoteUser, FEATURE.USE_EXPENSES)) {
+    return false;
+  } else {
+    return remoteUserMeetsOneCondition(req, expense, [isHostAdmin]);
```

Comment:
We do want to enable that for collective admins no?

```suggestion
    return remoteUserMeetsOneCondition(req, expense, [isHostAdmin, isCollectiveAdmin]);
```

Reply:
This morning, we decided to start with host admins only.